h1st/model/fuzzy/fuzzy_model.py

```diff
@@ -30,20 +30,20 @@ def __init__(self, variables: FuzzyVariables = None, rules: FuzzyRules = None):
                 skctrl.ControlSystem(rules.get_rules())
             )
 
-    def predict(self, input_data: Dict) -> Dict:
+    def execute_rules(self, input_data: Dict) -> Dict:
```

Comment:
@TgithubJ why don't we use the `predict` name to make it unify with other models?